launchable/commands/helper.py

```diff
@@ -106,40 +108,27 @@ def find_or_create_session(
                 "The build name you provided ({}) is different from the last build name recorded on this machine ({}).\n"
                 "Make sure to run `launchable record build --name {}` before you run this command.\n"
                 "If you already recorded this build on a different machine, use the --session option instead of --build. "
-                "See https://docs.launchableinc.com/sending-data-to-launchable/managing-complex-test-session-layouts".format(
+                "See https://www.launchableinc.com/docs/sending-data-to-launchable/using-the-launchable-cli/"
+                "recording-test-results-with-the-launchable-cli/managing-complex-test-session-layouts/".format(
                     build_name, saved_build_name, build_name), fg="yellow", ))
 
     session_id = read_session(saved_build_name)
     if session_id:
         _check_observation_mode_status(session_id, is_observation, tracking_client=tracking_client)
         return session_id
 
-    if build_name and saved_build_name != build_name:
-        raise click.UsageError(
-            click.style(
-                "The build name you provided ({}) is different from the last build name recorded on this machine ({}).\n"
-                "Make sure to run `launchable record build --name {}` before you run this command.\n"
-                "If you already recorded this build on a different machine, use the --session option instead of --build. "
-                "See https://docs.launchableinc.com/sending-data-to-launchable/managing-complex-test-session-layouts".format(
-                    build_name, saved_build_name, build_name), fg="yellow", ))
-
-    session_id = read_session(saved_build_name)
-    if session_id:
-        _check_observation_mode_status(session_id, is_observation, tracking_client=tracking_client)
-        return session_id
-    else:
-        context.invoke(
-            session_command,
-            build_name=saved_build_name,
-            save_session_file=True,
-            print_session=False,
-            flavor=flavor,
-            is_observation=is_observation,
-            links=links,
-            is_no_build=is_no_build,
-            lineage=lineage,
-        )
-        return read_session(saved_build_name)
```

Comment:
This code block exactly identical as the code right above, which makes me think this is a copy/paste error or merge error.